### ISSUE TITLE
ENH: Specify in the doc that geopandas.GeoSeries.areas doesn't compute the right area when coordinates are specified as lattitude/longitude 

### ISSUE THREAD

**Issue by gardiens** [geopandas/geopandas#3486]
#### Is your feature request related to a problem?
Hello, 
I wanted to use geopandas to compute areas on geopandas.Dataframe. 
However when I read the documentation [here](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.) It only specify that

>  "Area may be invalid for a geographic CRS using degrees as units; use [GeoSeries.to_crs()](https://geopandas.org/en/stable/docs/reference/api/geopandas.GeoSeries.to_crs.html#geopandas.GeoSeries.to_crs) to project geometries to a planar CRS before using this function.". 

However, when the polygon has coordinate with longitude latitude 
 ( even with crs  set)  instead of x y,  the code doesn't project the latitude longitude into a plane to compute the area . The resulting area is often very small   and does  not represent the true area.  
this  error is quite obnoxious because a usual user could assume that the coordinates are projected  into a plane if the crs is set. 
#### Describe the solution you'd like
I propose one solution: 

-  Add in the geopandas.GeoSeries.area note that " Area may be invalid if the geometry coordinates are expressed with longitude/lattitude".
-  It could point to some satisfactory [solution](https://gis.stackexchange.com/questions/413349/calculating-area-of-lat-lon-polygons-without-transformation-using-geopandas)

#### API breaking implications

I don't think the second solution would break anything, just specify information to the end users.
#### Describe alternatives you've considered

A solution which can be cumbersome and dangerous would be to add something that project back the coordinate into the plane to compute the right area, however it can be quite tricky and it depends on the use-case .

**Comment by m-richards** [comment]
Hi @gardiens, thanks for raising this. To me this is covered by the existing note you've mentioned as longitude and latitude coordinates defined in units of degrees. We could try and reword this to make it clearer?

Your point about a user assuming results might be automatically projected is fair if users are familiar with other tools with this behaviour. But geopandas does not do any automatic projection anywhere within the api - and we even warn explicitly about this in the situation you describe;
```python
from shapely import Polygon

import geopandas as gpd
p = Polygon([(0, 0), (1, 0), (1, 1)])
ser = gpd.GeoSeries([p], crs="EPSG:4326")
ser.area
UserWarning: Geometry is in a geographic CRS. Results from 'area' are likely incorrect. Use 'GeoSeries.to_crs()' to re-project geometries to a projected CRS before this operation.

  ser.area
```
Is there something we could have added to this warning that would have made this clearer for you?

**Comment by gardiens** [comment]
> Hi @gardiens, thanks for raising this. To me this is covered by the existing note you've mentioned as longitude and latitude coordinates defined in units of degrees. We could try and reword this to make it clearer?
> 



I think I didn't get my point right. My issue was working with that kind of samples from  government API : 

```
from shapely import Polygon

import geopandas as gpd
p = Polygon([(2.18, 2.1896), (2.189, 2.189), (2.1889, 2.1889)])
ser = gpd.GeoSeries([p], crs="EPSG:2154")
ser.area

4.800000e-07
```
In this sample : 
- the warning  is not displayed
- I checked the EPSG, here [this one ](https://epsg.io/2154). I saw that the unit is in meters, so I would have expected the results to be good. 


_However :_ 
I computed the area, and it was really absurd ( the true answers should have been around 6000 meters).

 In fact, the error was that the coordinates were in longitude/latitude and not in planar coordinates. 

I am not an expert in this field, but I think it may be a different problem from the warning you displayed. That's why I proposed to add a note in the documentation, but a rephrasing of some parts could be even better 💯

**Comment by martinfleis** [comment]
I am sorry but this is a clear case of a user error. This line

```py
ser = gpd.GeoSeries([p], crs="EPSG:2154")
```

passes a polygon with coordinates using latitude and longitude but does not assign a projection that would indicate that (that would be `EPSG:4326`) but, probably by mistake, `EPSG:2154`. There's no way for geopandas to tell that this is wrong and from this point on it assumes EPSG:2154, which is projected and uses meters and does not warn. Because from the geopandas perspective, all is good and the value is correct. 

You need to ensure that the coordinates match the crs assigned to them, otherwise you can get in a lot of trouble with many functions. The ideal situation in your case would look like this:

```py
from shapely import Polygon

import geopandas as gpd
p = Polygon([(2.18, 2.1896), (2.189, 2.189), (2.1889, 2.1889)])
ser = gpd.GeoSeries([p], crs="EPSG:4326").to_crs("ESPG:2154")
ser.area

0    10006.695009
dtype: float64
```
